src/Test.UnitTests.Sarif.Multitool.Library/ApplyPolicyCommandTests.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using FluentAssertions;
+using Moq;
+using System.IO;
```

Comment:
General stylistic pattern is to have all System usings before the rest then alpha after.

Reply:
done!